src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -59,23 +59,38 @@ public function getLocator()
     /**
      * Imports a resource.
      *
-     * @param mixed       $resource       A Resource
-     * @param string|null $type           The resource type or null if unknown
-     * @param bool        $ignoreErrors   Whether to ignore import errors or not
-     * @param string|null $sourceResource The original resource importing the new resource
+     * @param mixed                $resource       A Resource
+     * @param string|null          $type           The resource type or null if unknown
+     * @param bool                 $ignoreErrors   Whether to ignore import errors or not
+     * @param string|null          $sourceResource The original resource importing the new resource
+     * @param string|string[]|null $exclude        A globbed path of files to exclude or an array of globbed paths of files to exclude
      *
      * @return mixed
      *
      * @throws LoaderLoadException
      * @throws FileLoaderImportCircularReferenceException
      * @throws FileLocatorFileNotFoundException
      */
-    public function import($resource, $type = null, $ignoreErrors = false, $sourceResource = null)
+    public function import($resource, $type = null, $ignoreErrors = false, $sourceResource = null, $exclude = null)
```

Comment:
Adding an argument to a public method in a non-final class is a BC break. 
See https://symfony.com/doc/current/contributing/code/bc.html#changing-classes, `Add argument with a default value`

Reply:
Do I need to achieve that without a BC break ? If so, how can I achieve that ?

Reply:
You can add the argument as commented `/* , $exclude = null */` and switch behavior based on `if (5 === func_num_args())`. You also have to add a deprecation notice. See [DebugProcessor](https://github.com/symfony/symfony/blob/63d730920b43ee5355c6ff3a730fc67a8c325db7/src/Symfony/Bridge/Monolog/Processor/DebugProcessor.php#L64) as an example.